cumulus/parachains/common/src/impls.rs

```diff
@@ -35,28 +36,33 @@ pub type AccountIdOf<R> = <R as frame_system::Config>::AccountId;
 
 /// Implementation of `OnUnbalanced` that deposits the fees into a staking pot for later payout.
 pub struct ToStakingPot<R>(PhantomData<R>);
-impl<R> OnUnbalanced<NegativeImbalance<R>> for ToStakingPot<R>
+impl<R> OnUnbalanced<fungible::Credit<R::AccountId, pallet_balances::Pallet<R>>> for ToStakingPot<R>
```

Comment:
there is a generic implementations for fungible and fungibles, they do exactly the same 
https://github.com/paritytech/polkadot-sdk/blob/02e8061bbadf51bce2f8c419c8645555532c7489/substrate/frame/support/src/traits/tokens/imbalance/on_unbalanced.rs#L63

Reply:
Cool, so we can completely remove this?

Reply:
Hadn't noticed this, thanks